docs/collector-limitations.md

```diff
@@ -0,0 +1,8 @@
+# Elastic OpenTelemetry Collector limitations
+
+The Elastic Distribution for the OpenTelemetry Collector has the following limitations:
+
+- `host.network.*` metrics aren't present from the OpenTelemetry side.
+- `process.state` isn't present in the OpenTelemetry host metric. It's set to a dummy value of **Unknown** in the **State** column of the host processes table.

```

Comment:
@ishleenk17 is this an upstream limitation?

Reply:
The process.state is not present in the Hostmetrics receiver. So, yes its an upstream [limitation](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33731). 
At Elastic, we show the state as Unknown for all processes because of this limitation.